marlowe-playground-client/src/Halogen/ActusBlockly.purs

```diff
@@ -16,16 +16,26 @@ import Data.Symbol (SProxy(..))
 import Effect.Class (class MonadEffect)
 import Halogen (ClassName(..), Component, HalogenM, RefLabel(..), liftEffect, mkComponent, raise)
 import Halogen as H
-import Halogen.HTML (HTML, button, div, text)
+import Halogen.HTML (HTML, button, div, text, iframe, aside, section)
```

Comment:
Did you run `purty`? It should have sorted the imports, I guess.

Reply:
I usually run those after review - coz my nix is on separate machine